src/wp-includes/class-wp-query.php

```diff
@@ -1967,6 +1972,12 @@ public function get_posts() {
 			$q['update_post_meta_cache'] = true;
 		}
 
+		if ( ! isset( $q['lazy_load_post_meta'] ) ) {
+			$q['lazy_load_post_meta'] = false;
```

Comment:
Per the DocBlock, shouldn't this be the following?

```suggestion
			$q['lazy_load_post_meta'] = $q['update_post_meta_cache'];
```

Reply:
Rather than just removing the docs about the default in [4e3a8ec](https://github.com/WordPress/wordpress-develop/pull/4216/commits/4e3a8ecbb8a172501cefe184ca413d929fe4239a), can it be updated to be accurate?

Reply:
@joemcgill I am not sure what is being asked here.